src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -622,20 +622,23 @@ public virtual IEnumerable<CallStackFrame> GetCallStack()
         /// Get a breakpoint by id, primarily for Enable/Disable/Remove-PSBreakpoint cmdlets.
```

Comment:
@TylerLeonhardt, your last commit had 9 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can set command breakpoints
```powershell
Exception calling "SetCommandBreakpoint" with "2" argument(s): "The background process reported an error with the following message: Remote transport error: ."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 81
81:             $jobRunspace.Debugger.SetCommandBreakpoint('Write-Verbose', { break }) | Should -BeOfType [System.Management.Automation.CommandBreakpoint]

```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can set variable breakpoints
```powershell
Exception calling "SetVariableBreakpoint" with "3" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 85
85:             $jobRunspace.Debugger.SetVariableBreakpoint('DebugPreference', 'ReadWrite', { break }) | Should -BeOfType [System.Management.Automation.VariableBreakpoint]

```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can set line breakpoints
```powershell
Exception calling "SetLineBreakpoint" with "4" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 89
89:             $jobRunspace.Debugger.SetLineBreakpoint($PSCommandPath, 1, 1, { break }) | Should -BeOfType [System.Management.Automation.LineBreakpoint]

```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can get breakpoints
```powershell
Exception calling "GetBreakpoints" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 94
94:             $jobRunspace.Debugger.GetBreakpoints() | Should -HaveCount 4

```
Breakpoint SDK Unit Tests.Managing breakpoints in a remote runspace via the SDK.Can disable breakpoints
```powershell
Exception calling "GetBreakpoints" with "0" argument(s): "The runspace state is not valid for this operation."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1: line 98
98:             foreach ($bp in $jobRunspace.Debugger.GetBreakpoints()) {

```